ebodac/src/test/java/org/motechproject/ebodac/uitest/test/HiddenButtonsEnabledUiTest.java

```diff
@@ -0,0 +1,51 @@
+package org.motechproject.ebodac.uitest.test;
+
+import org.apache.commons.lang.StringUtils;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.motech.page.LoginPage;
+import org.motech.test.TestBase;
+import org.motechproject.ebodac.uitest.page.HomePage;
+import org.motechproject.ebodac.uitest.page.ParticipantEditPage;
+import org.motechproject.ebodac.uitest.page.ParticipantPage;
+
+import static org.junit.Assert.assertTrue;
+
+public class HiddenButtonsEnabledUiTest extends TestBase {
+
+    private LoginPage loginPage;
+    private HomePage homePage;
+    private String l1adminUser;
+    private String l1adminPassword;
+    private ParticipantPage participantPage;
+    private ParticipantEditPage participantEditPage;
+
+    @Before
+    public void setUp() throws Exception {
+        l1adminUser = "admin";
+        l1adminPassword = System.getProperty("admin_password");
+        loginPage = new LoginPage(driver);
+        homePage = new HomePage(driver);
+        participantPage = new ParticipantPage(driver);
+        participantEditPage = new ParticipantEditPage(driver);
+        if (StringUtils.equals(homePage.expectedUrlPath(), currentPage().urlPath())) {
```

Comment:
it was "!=" before and now you have equals, should not this be !StringUtils.equals?